backend/src/main/java/com/google/blackswan/mock/SimpleRelatedDataGenerator.java

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.blackswan.mock;
+
+import com.google.models.*;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.HashMap;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.ArrayListMultimap;
+
+/** Singleton class to generate related data for a given anomaly. */
+public class SimpleRelatedDataGenerator implements RelatedDataGenerator {
+  private static final String TARGET_DIMENSION = "Ramen";
+  private static final String RELATED_METRIC = "Interest Over Time";
+  private static final String CONFIG_USERNAME = "catyu@";
+  private static final ImmutableMap<String, String> FILE_LOCATION = ImmutableMap.of(
+                                                                      "Udon", "/udon-data.csv",
+                                                                      "Pho", "/pho-data.csv"
+                                                                    );
+
+  private static final SimpleRelatedDataGenerator INSTANCE = new SimpleRelatedDataGenerator();
+
+  private Multimap<String, DataInfo> relatedDataMap;
```

Comment:
is this is a map from anomaly dimension name to related data info?
from my understanding, it seems like this is like the result of processing the configs and then putting it in a data structure that makes it easy to get the actual related data?

Reply:
yes, it is currently map from anomaly dimension name to related data info, but in the future it should also include anomaly metric name to its related data.

yes, it should be the result of processing the configs from the datastore :) this map makes it quick to determine whether there are related data for any dimension/metric name and if so, what those related data are.